packages/flutter/test/cupertino/scaffold_test.dart

```diff
@@ -25,6 +25,31 @@ void main() {
     expect(tester.getTopLeft(find.byType(Center)), const Offset(0.0, 0.0));
   });
 
+testWidgets('Opaque bar pushes contents down', (WidgetTester tester) async {
```

Comment:
There's already a test below that checks that the content is under the opaque bar. Here specifically call out that the inner content sees no top media query padding.

Reply:
'Child sees no media query padding when below opaque nav bar'